### ISSUE TITLE
Next 9.5.3 -> 9.5.4 breaks local font resolution

### ISSUE THREAD

**Issue by timfee** [vercel/next.js#17756]
# Bug report

## Describe the bug

I'm loading a custom font:
```css
/* ./src/css/main.css */

@font-face {
  font-family: 'Space Grotesk';
  src: url('/fonts/SpaceGrotesk-Bold.woff2') format('woff2'),
    url('/fonts/SpaceGrotesk-Bold.woff') format('woff');
  font-weight: bold;
  font-style: normal;
  font-display: swap;
}
```

And I have the fonts living in `./public/fonts`

## To Reproduce

Using 9.5.3, I was able to build the app successfully ([code](https://github.com/timfee/homepage/tree/2244b85a73bb78c9e48d3932bd2473a2270be697))

With the only change being 9.5.4, I can no longer build ([diff/code](https://github.com/timfee/homepage/commit/a2464ebb425d6b7fd9dd5bdb1081464676ef380a))

## Expected behavior

Continues to work :)

## Screenshots

The stdout error:

```
error - ./src/css/fonts.css (./node_modules/css-loader/dist/cjs.js??ref--5-oneOf-6-1!./node_modules/next/dist/compiled/postcss-loader??__nextjs_postcss!./src/css/fonts.css)
Error: Can't resolve '/fonts/SpaceGrotesk-Bold.woff2' in '/Users/timfee/Developer/homepage/src/css'
```

## System information

- OS: macOS **Big Sur 11.0 Beta (20A5384c)**
- Version of Next.js: 9.5.4
- Version of Node.js: v14.11.0

## Additional Information

If I specify a relative path in `main.css` (e.g. `url('../../public/[font].woff')`), then it gets picked up by Webpack (but breaks my preload)

**Comment by brothatru** [comment]
Same here.

Also happening for image urls:

```bash
Failed to compile.

./components/templates/SignUpPageTemplate.module.scss
Error: Can't resolve '`/compressed/static/images/img-color-arrows.png`' in '/usr/src/app/components/templates'
    at runMicrotasks (<anonymous>)

```

**Comment by ijjk** [comment]
Hi, this is a duplicate of https://github.com/vercel/next.js/issues/17701 and should be corrected in `v9.5.5-canary.0` of Next.js.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
